snuba/datasets/processors/spans_processor.py

```diff
@@ -382,6 +381,11 @@ def process_message(
             processed_rows.append(processed)
             processed_spans = self._process_spans(event_dict, common_span_fields)
             processed_rows.extend(processed_spans)
+
+            # extract platform from spans
+            if processed_spans:
+                processed["platform"] = processed_spans[0].get("platform", "")
```

Comment:
`processed` dictionary is not useful here since all processed spans have already been added to the `processed_rows` before this. For the root span, I would recommend that you set the `platform` to empty string in `_process_module_details`. When you make that change, the tests would change to reflect that `platform` is set to empty string in the first span of the result

Reply:
For now yes. But it is something we should follow up on. I can initiate the conversation to figure it out and tag you

Reply:
ok I set to empty for now.